modules/graphstudio/pages/explore-graph/actions.adoc

```diff
@@ -0,0 +1,38 @@
+= Actions
+
+The Actions tab in GraphStudio is a hub for actions that affect the entire graph database.
+
```

Comment:
We should mention that all functions in this page can only be executed in Global View.

Reply:
Agree that we should call out that global view needs calling out, but not sure we need to add screenshots. Screenshots tend to clutter the page and are hard to maintain, so we try to avoid using them.